### ISSUE TITLE
Tab bar animation issue when pop to view controller interactively 

### ISSUE THREAD

**Issue by iDevelopper** [LeoNatan/LNPopupController#318]
#### Description
The tab bar is not animated properly when come back from next interactively and when the popup bar is not presented.

#### Steps to Reproduce
In FirstViewController.m, do not present the popup bar,
Launch app, touch Tab Bar Controller + Navigation Controller, then Next.
Swipe Left to Right to come back.

![simulator screen shot - iphone x - 2019-01-10 at 12 16 34](https://user-images.githubusercontent.com/1782450/50965883-9c08fb00-14d3-11e9-90d9-c872831f92e4.png)

#### Device, OS and Xcode Versions
Xcode 10.1, iOS 12.1 simulator

**Comment by iDevelopper** [comment]
Seems this is an Apple bug. When I comment the swizzle methods, get the same issue:
```
		//_hideBarWithTransition:isExplicit:
        /*
		selName = _LNPopupDecodeBase64String(hBWTiEBase64);
		__swizzleInstanceMethod(self,
								NSSelectorFromString(selName),
								@selector(hBWT:iE:));
		
		//_showBarWithTransition:isExplicit:
		selName = _LNPopupDecodeBase64String(sBWTiEBase64);
        
		__swizzleInstanceMethod(self,
								NSSelectorFromString(selName),
								@selector(sBWT:iE:));
		*/
```
And testing a sample project give me also the same issue:
[TestHidesTabBarWhenPushed.zip](https://github.com/LeoNatan/LNPopupController/files/2745282/TestHidesTabBarWhenPushed.zip)

**Comment by LeoNatan** [comment]
This is an Apple bug that was fixed in 12.1.x but because the simulator runtime has not been updated, it still shows it. Try running on a real device.

This bug happened to WhatsApp, for example, because they have the same gesture.

**Comment by iDevelopper** [comment]
Yes, I have already test it on a 12.1.3 device. Solved.
